packages/hardhat/test/e2e/periphery/controller-helper.ts

```diff
@@ -1356,4 +1460,635 @@ it("Close vault LP and open new one-siced LP with just oSQTH ", async () => {
+          amount1Min: 0,
+          deadline: Math.floor(await getNow(ethers.provider) + 8640000),
+        })
+        const wPowerPerpAmountInLPAfter = (isWethToken0) ? amount1After : amount0After;
+        const wethAmountInLPAfter = (isWethToken0) ? amount0After : amount1After;
+        // Changes
+        const depositorEthDiff = depositorEthBalanceAfter.sub(depositorEthBalanceBefore)
+        const depositorSqueethDiff = depositorSqueethBalanceAfter.sub(depositorSqueethBalanceBefore)
+        const vaultEthDiff = vaultAfter.collateralAmount.sub(vaultBefore.collateralAmount)
+        const vaultSqueethDiff = vaultAfter.shortAmount.sub(vaultBefore.shortAmount)
+        const lpEthDiff = wethAmountInLPAfter.sub(wethAmountInLPBefore)
+        const lpSqueethDiff = wPowerPerpAmountInLPAfter.sub(wPowerPerpAmountInLPBefore)
+        // Assertions
+        // console.log('currentTick', currentTick)
+        console.log('positionAfter.TickLower', positionAfter.tickLower)
+        console.log('positionAfter.TickUpper', positionAfter.tickUpper)
+        console.log('wPowerPerpAmountInLPBefore', wPowerPerpAmountInLPBefore.toString())
+        console.log('wethAmountInLPBefore', wethAmountInLPBefore.toString())
+        console.log('wPowerPerpAmountInLPAfter', wPowerPerpAmountInLPAfter.toString())
+        console.log('wethAmountInLPAfter', wethAmountInLPAfter.toString())
+        console.log('vaultSqueethDiff',vaultSqueethDiff.toString() )
+        console.log('vaultEthDiff',vaultEthDiff.toString() )
+        console.log('lpSqueethDiff', lpSqueethDiff.toString())
+        console.log('lpEthDiff', lpEthDiff.toString())
+        console.log('depositorSqueethDiff', depositorSqueethDiff.toString())
+        console.log('vaultBefore.collateralAmount', vaultBefore.collateralAmount.toString())
+        console.log('vaultBefore.shortAmount', vaultBefore.shortAmount.toString())
+        console.log('vaultAfter.collateralAmount', vaultAfter.collateralAmount.toString())
+        console.log('vaultAfter.shortAmount', vaultAfter.shortAmount.toString())
+        console.log('wethAmountToLp',wethAmountToLp.toString())
+        console.log('flashloanAmount',flashLoanAmount.toString())
+        console.log('wPowerPerpAmountInLPBefore', wPowerPerpAmountInLPBefore.toString())
+        console.log('wPowerPerpAmountToLp', wPowerPerpAmountToLp.toString())
+        console.log('depositorEthDiff', depositorEthDiff.toString())
+        console.log('depositorSqueethDiff', depositorSqueethDiff.toString())
+        console.log('gasSpent', gasSpent.toString())
+    
+        // expect(positionAfter.tickLower === newTickLower).to.be.true
+        expect(vaultSqueethDiff.sub(wPowerPerpAmountToLp).eq(0)).to.be.true
+        // Nft id unchanged
+        expect(vaultAfter.NftCollateralId==vaultBefore.NftCollateralId).to.be.true
+        // Squeeth convervation
+        expect(lpSqueethDiff.sub(vaultSqueethDiff).add(depositorSqueethDiff).abs().lte(10)).to.be.true
```

Comment:
prb should check the ETH amounts as well

Reply:
added         `expect((vaultEthDiff.add(wethAmountToWithdraw)).abs().lte(10)).to.be.true`